templates/unit-tests-with.yml

```diff
@@ -37,23 +47,29 @@ jobs:
       versionSpec: ${{ parameters.python_version }}
     displayName: Use Python ${{ parameters.python_version }}
 
-  # Install all Toolkit dependencies for this repo.
-  # TODO: Maybe look inside info.yml to find which frameworks need to be cloned?
-  - bash: git clone https://github.com/shotgunsoftware/tk-core --depth 1 --branch SG-13264-python-3-compat ../tk-core
-    displayName: Installing Toolkit Dependencies
+  - template: pip-install-packages.yml
+    parameters:
+      packages:
+      - ${{ parameters.qt_wrapper }}
+      - git+https://github.com/shotgunsoftware/tk-toolchain.git@${{ parameters.tk_toolchain_ref }}#egg=tk-toolchain
+      - pytest-azurepipelines
+      - ${{ parameters.extra_test_dependencies }}
```

Comment:
Does this explode the list somehow?  I'm probably just not understanding how it works, but it looks like pip-install-packages will try and do `pip install ${{ package }}` for this list, rather than for items in it?

Reply:
(Obviously the CI is succeeding and Qt.py is being installed, so this is working, I guess I just am wondering if having multiple extra_test_dependencies will work as expected.)

Reply:
Ah yes, very good question! It indeeds works that way! When a template parameter that is a list is inserted into another list it extends the destination array instead of being nested. I'll add a commit about it.